cluster/gce/util.sh

```diff
@@ -1344,11 +1349,7 @@ function ssh-to-node {
 
 # Restart the kube-proxy on a node ($1)
 function restart-kube-proxy {
-  if [[ "${OS_DISTRIBUTION}" == "trusty" ]]; then
-    ssh-to-node "$1" "sudo initctl restart kube-proxy"
-  else
-    ssh-to-node "$1" "sudo /etc/init.d/kube-proxy restart"
-  fi
+  ssh-to-node "$1" "sudo /etc/init.d/kube-proxy restart"
```

Comment:
With kube-proxy in a pod, there is no longer anything in `/etc/init.d/kube-proxy` so what you've left isn't useful either. I'd just remove this change since it isn't related to what this PR is trying to do and is covered by my other change (and then they won't have conflicts which will make it easier to merge both and get them cherry picked).